arlo_server/__init__.py

```diff
@@ -315,6 +316,55 @@ def election_new():
     return jsonify(electionId=election_id)
 
 
+@app.route('/election/<election_id>/jurisdictions_file', methods=["GET"])
```

Comment:
I'm still wrapping my head around the API design a bit. Wondering if this should be part of the `/election/{id}/jurisdictions` API or if it's a separate API like you have it here.

Reply:
I mostly just followed @benadida's suggestion from #303, though I think having it in `…/jurisdictions` could be a little confusing. We've been discussing with @mcchilders that elections officials have a mental model that includes "files" for things like the list of jurisdictions being the source of truth, and so we're trying to model that here.